app/vmui/packages/vmui/src/pages/ExploreAlerts/ExploreRules.tsx

```diff
@@ -33,16 +33,13 @@ const ExploreRules: FC = () => {
   const [modalOpen, setModalOpen] = useState(true);
   const [searchParams, setSearchParams] = useSearchParams();
 
-  const navigate = useNavigate();
-  const location = useLocation();
-
   useEffect(() => {
-    if (!location.hash && groupId) {
+    if (groupId) {
       setModalOpen(true);
     } else {
       setModalOpen(false);
     }
```

Comment:
This can be simplified to:
```suggestion
    setModalOpen(!!groupId)
```